processor/src/utils.rs

```diff
@@ -0,0 +1,41 @@
+use super::{Felt, StarkField, Vec};
+
+// HELPER FUNCTIONS
+// ================================================================================================
+
+/// Returns the number of rows in the provided execution trace assumed to be in column-major form
+/// and contain at least one column.
+pub fn get_trace_len(trace: &[Vec<Felt>]) -> usize {
+    trace[0].len()
+}
+
+/// Splits an element into two field elements containing 32-bit integer values
+#[inline(always)]
+pub fn split_element(value: Felt) -> (Felt, Felt) {
+    let value = value.as_int();
+    let lo = (value as u32) as u64;
+    let hi = value >> 32;
+    (Felt::new(hi), Felt::new(lo))
+}
+
+/// Splits an element into two 16 bit integer limbs. It assumes that the field element contains a
+/// valid 32-bit integer value.
+pub fn split_element_u32_into_u16(value: Felt) -> (Felt, Felt) {
+    let (hi, lo) = split_u32_into_u16(value.as_int());
+    (Felt::new(hi as u64), Felt::new(lo as u64))
+}
+
+/// Splits a u64 integer assumed to contain a 32-bit value into two u64 integers containing 16-bit
+/// values.
```

Comment:
small typo: "... splits into two u16 integers..."

Reply:
Thanks! I guess I updated the signature & forgot to update the comment, oops.